pages/en/lb4/Getting-started.md

```diff
@@ -24,28 +24,77 @@ The starting point is [ECMAScript 6](http://www.ecma-international.org/ecma-262/
 - Running the app (node ., npm start)
 " %}
 
+## Installation
+
+Make sure you have the following installed:
+
+- Node.js >= 7.0.0
+- TypeScript >= 2.0.0 `npm i -g typescript`
+- TypeScript Node >= 3.0.0 `npm i -g ts-node`
+
 ## A simple example
 
-Here is the most basic LoopBack 4 application:
+This example creates an `Application` that responds to all HTTP requests with the text "Hello World".
+
+### Building a LoopBack 4 application
+First create a new folder.  Then, we are going to create 2 files within this folder: `index.ts` and `tsconfig.json`.  
 
 ```js
+//index.ts
 import {Application} from '@loopback/core';
+import {RestComponent, RestServer} from '@loopback/rest';
 
-const app = new Application();
-app.handler((sequence, request, response) => {
-  sequence.send(response, 'hello world');
-});
-
-(async function start() {
-  await app.start();
-  console.log(`The app is running on port ${app.getSync('http.port')}`);
+export class HelloWorldApp extends Application {
+    constructor() {
+      super({
+        components: [RestComponent]
+      });
+    }
+  
+    async start() {
+      const rest = await this.getServer(RestServer);
+      rest.handler((sequence, request, response) => {
+        sequence.send(response, 'Hello World');
+      });
+      await super.start();
+      console.log(`REST server running on port: ${await rest.get('rest.port')}`);
+    }
+  }
+  
+(async function main() {
+    const app = new HelloWorldApp();
+    await app.start();
 })();
+
+```
+
+Next, create a file called `tsconfig.json`.  `tsconfig.json` is needed to indicate that this is a TypeScript project.  It also includes the compiler options which are required for compiling this application.  For more details, see [TypeScript documentation](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html).
 ```
+//tsconfig.json
+{
+    "compilerOptions": {
+      /* Basic Options */
+      "target": "es6",                          
+      "module": "commonjs",                     
 
-The example above creates an `Application` that responds to all HTTP requests with the text "Hello World".
+      /* Experimental Options */
+      "experimentalDecorators": true,
+      "emitDecoratorMetadata": true
+    }
+  }
+```
+### Running the application
 
-To see what the complete application looks like, see [loopback-next-hello-world](https://github.com/strongloop/loopback-next-hello-world/).
+To install the required packages, run:
+```
+npm install --save @loopback/core @loopback/rest
+```
+
+To run the application, run:
+```
+ts-node index.ts
+```
 
-The following guides introduce the fundamental concepts of LoopBack 4 required to build scalable, maintainable, and consistent APIs.
+## References
 
-If you want to try it out locally follow the instructions in [Installation](Installation.html).  
+For more examples and tutorials, see [http://loopback.io/doc/en/lb4/Examples-and-tutorials.html](http://loopback.io/doc/en/lb4/Examples-and-tutorials.html). 
```

Comment:
Use relative URL to avoid displaying the "external site" icon.
```
For more examples and tutorials, see [Examples-and-tutorials.html](http://loopback.io/doc/en/lb4/Examples-and-tutorials.html).
```